MantidQt/CustomInterfaces/test/EnggDiffractionViewMock.h

```diff
@@ -143,11 +143,22 @@ class MockEnggDiffractionView
   // void plotRepChanged();
   MOCK_METHOD1(plotRepChanged, void(int idx));
 
+  // virtual void plotFocusedSpectrum();
+  MOCK_METHOD1(plotFocusedSpectrum, void(const std::string &));
+
   // virtual void plotWaterfallSpectrum
   MOCK_METHOD1(plotWaterfallSpectrum, void(const std::string &wsName));
 
   // virtual void plotReplacingWindow
   MOCK_METHOD1(plotReplacingWindow, void(const std::string &wsName));
+
+  // virtual void plotVanCurvesCalibOutput();
+  MOCK_METHOD0(plotVanCurvesCalibOutput, void());
+
+  // virtual void plotDifcZeroCalibOutput();
+  MOCK_METHOD3(plotDifcZeroCalibOutput,
+               void(std::vector<double> &difc, std::vector<double> &tzero,
+                    std::string &specNo));
```

Comment:
This adds the mock methods, but there are no changes to the tests. We don't have any full positive test where we can test the actual plots (plus that would require the python interpreter and graphical environment), but some of the tests could do a simple check that some of these plot methods are not called when they should not be called. Just a simple basic checks that the logic behind the plots is not broken.